packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -288,7 +288,7 @@ class FlutterError extends AssertionError {
         String message = details.exceptionAsString();
         if (message.startsWith(prefix))
           message = message.substring(prefix.length);
-        debugPrint('The following $errorName was $verb:\n$message', wrapWidth: wrapWidth);
+        debugPrint('\nThe following $errorName was $verb:\n\n$message', wrapWidth: wrapWidth);
```

Comment:
Is the leading one here useful? There's a horizontal line immediately above this line, so it's not something that could get confused with another paragraph.